lib/charts/data_transformers/aggregation.dart

```diff
@@ -102,10 +104,10 @@ class AggregationModel {
   /// Create a new [AggregationModel] from a [collection] of items,
   /// list of [dimensions] on which the items are grouped and a list of [facts]
   /// on which aggregations are computed.
-  AggregationModel(List collection, List dimensions, List facts,
+  AggregationModel(Iterable collection, List dimensions, List facts,
```

Comment:
Because of this line basically:
https://github.com/google/charted/blob/master/lib/charts/data_transformers/aggregation_transformer.dart#L77

ChartData.rows is an Iterable